### PR TITLE
Update to use 2020 project

### LINKED ISSUE #67: 2020 Support
The build template files need to be updated for 2020.

### PR DESCRIPTION
**Pull request by Daltz333**
Needs to be updated for release, and needs testing.

Closes #67

### CODE REVIEW

File: frc_characterization/robot/project/vendordeps/REVRobotics.json [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+            "binaryPlatforms": [
+                "windowsx86-64",
+                "windowsx86",
+                "linuxx86-64",
+                "linuxathena",
+                "linuxraspbian"
+            ]
+        }
+    ]
+}
```
Comment (AustinShalit) [inline frc_characterization/robot/project/vendordeps/REVRobotics.json]: eol @ eof

**Comment by sciencewhiz** [comment]
Shouldn't the gradle version be 6.0.1?

**Comment by Daltz333** [comment]
It wasn't in the GradleRIO example projects.

**Comment by sciencewhiz** [comment]
In GradleRIO the distributionURL is 6.0.1, so that's what version it downloads.

https://github.com/wpilibsuite/GradleRIO/blob/5706a8397343a2c156a186af9034253828ca0151/examples/cpp/gradle/wrapper/gradle-wrapper.properties#L3

**Pull request merged**